src/aws-infra.ts

```diff
@@ -68,7 +67,14 @@ class UKCoronavirusDataAlertsStack extends GuStack {
         })
     }
 }
+export const account = "745349931791";
```

Comment:
why do we need this hardcoded? I remember Adam Fisher having some thing about not hardcoding account numbers, though I never fully understood the rationale (I think it was defense in depth)

Reply:
Good spot - this is my laziness copying stuff over from Lurch - I think in Lurch we use the account number to do some kind of VPC lookup thing (or we did when we first created it and haven't updated it since) - but this app doesn't need either of the environment properties I'm passing in